core/identity/identity.go

```diff
@@ -373,17 +382,17 @@ func (s *service) observeIdentitiesLoop() {
 		case <-s.closing:
 			return
 		case <-s.identityForceUpdate:
-			observe()
+			observe(true)
 		case <-ticker.C:
-			observe()
+			observe(false)
 		}
 	}
 }
 
 const identityRepoDataKind = "profile"
 
 // TODO Maybe we need to use backoff in case of error from coordinator
-func (s *service) observeIdentities(ctx context.Context) error {
+func (s *service) observeIdentities(ctx context.Context, forceUpdate bool) error {
```

Comment:
not just forceUpdate, but globalNameForceUpdate

Reply:
Done